compute-budget/src/compute_budget_limits.rs

```diff
@@ -36,13 +40,25 @@ impl Default for ComputeBudgetLimits {
     }
 }
 
+//#[cfg(test)]
+pub fn get_prioritization_fee_for_test(compute_unit_price: u64, compute_unit_limit: u64) -> u64 {
+    get_prioritization_fee(compute_unit_price, compute_unit_limit)
+}
+
+fn get_prioritization_fee(compute_unit_price: u64, compute_unit_limit: u64) -> u64 {
```

Comment:
I think better to just make this public for dcou instead of having a for_tests wrapper

Reply:
> be made as public **only** for dev-context-utils-only.

Yea, this sounds reasonable. 

Apologize for late inputs on this, but ideally I'd prefer to keep this new `fn get_prioritization_fee()` private, or just inline into `from()`. The reason is to try to keep `compute_budget_limits.rs` as just-a-struct as possible, so it's easier to remove in the near future (say Transaction v1).

The tests of `bank` can be updated to getting prio_fee from FeeBudgetLimits, without having to call `get_prioritization_fee()`, maybe something something like this: 
```
    let fee_budget_limits = FeeBudgetLimits::from(
        ComputeBudgetLimits {
            compute_unit_price: xyz,
            compute_unit_limit: abc,
            ..ComputeBudgetLimits::defafult(),
        },
    );
    let prio_fee = fee_budget_limits.prioritization_fee;
    ... ...
```